go/libkb/sig_chain.go

```diff
@@ -490,6 +490,13 @@ func (sc *SigChain) verifySubchain(ctx context.Context, kf KeyFamily, links []*C
 			}
 		}
 
+		if pgprev, ok := tcl.(*RevokeChainLink); ok {
+			for _, revokedKid := range pgprev.GetRevokeKids() {
```

Comment:
This isn't sufficient since we can get PGP in non-revoke chain links. The right place to handle this is `ComputedKeyFamily#RevokeSig` and `ComputedKeyFamily#RevokeKid`.  It's all right there for you.